src/sidebar/components/search-status-bar.js

```diff
@@ -21,16 +21,14 @@ const countVisibleAnns = annThread => {
  * A bar where the user can clear a selection or search and see whether
  * any search results were found.
  * */
-function SearchStatusBar({
-  selectedTab,
-  totalAnnotations,
-  totalNotes,
-  rootThread,
-}) {
+function SearchStatusBar({ rootThread }) {
   const storeState = useStore(store => store.getState());
   const clearSelection = useStore(store => store.clearSelection);
   const filterQuery = storeState.filterQuery;
   const filterActive = !!storeState.filterQuery;
+  const countAnnotations = useStore(store => store.countAnnotations());
```

Comment:
These variable names confused me because they read as actions rather than values. `annotationCount` and `noteCount` would be clearer.